ablog/start.py

```diff
@@ -623,9 +620,10 @@ def ask_user(d):
                 'be generated relative to this URL. If you don\'t have one yet, '
                 'you can set it in configuration file later.'))
         if SPHINX_LT_17:
+            # APR: Not sure how do_prompt() worked prior to Sphinx 1.7; likely to be `lambda x: x` here too
```

Comment:
What if we bumped up the min Sphinx version to 1.7, would that make life easier?

Reply:
@nabobalis It would make development easier; but I'm not sure where the user base is at. There's a few upgrades that would be helpful:

1. The modern version of Sphinx requires Python 3.5+. They dropped support for Python 2.7 and 3.4 with the 2.x release. 
2. Python 2.7 goes end-of-life in six months or so. 
3. The Sphinx 1.7 checks that are in the codebase. 
4. Sphinx 3.0 is on the horizon, which drops some support for items deprecated in Sphinx 1.8. 

At some point, jumping to Sphinx 2.x and Python 3.x+ would make life easier on development. I don't have a good sense on when the timing of that should be, though.

Reply:
Ok. I should probably bump the min version and drop python 2 and release a new major version of the package.